src/overlays/actors/ovl_En_Dekunuts/z_en_dekunuts.c

```diff
@@ -71,85 +121,634 @@ static DamageTable D_808BEF64 = {
+        if (limbIndex == 5) {
+            currentFrame = this->skelAnime.animCurrentFrame;
+            if (currentFrame <= 6.0f) {
+                arg2 = 1.0f - (currentFrame * 0.0833f);
+                arg1 = (currentFrame * 0.1167f) + 1.0f;
+                arg3 = (currentFrame * 0.1167f) + 1.0f;
+            } else if (currentFrame <= 7.0f) {
+                currentFrame -= 6.0f;
+                arg2 = 0.5f + currentFrame;
+                arg1 = 1.7f - (currentFrame * 0.7f);
+                arg3 = 1.7f - (currentFrame * 0.7f);
+            } else if (currentFrame <= 10.0f) {
+                arg2 = 1.5f - ((currentFrame - 7.0f) * 0.1667f);
+                arg1 = 1.0f;
+                arg3 = 1.0f;
+            } else {
+                return 0;
+            }
+            Matrix_Scale(arg1, arg2, arg3, MTXMODE_APPLY);
+        } else if ((limbIndex == 2) && (this->actor.params == 2)) {
+            rot->z = this->actor.world.rot.x;
+        }
+    }
+    return 0;
+}
+
+void func_808BED30(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    static s8 D_808BEF98[] = {
+        0xFF, 0xFF, 0xFF, 0x03, 0xFF, 0x00, 0xFF, 0x01, 0xFF, 0x02, 0x00, 0x00,
+    };
+
+    static Vec3f D_808BEFA4[] = {
+        { -1500.0f, 0.0f, -1700.0f }, { -1500.0f, 0.0f, 1700.0f }, { -2500.0f, -2000.0f, 0.0f },
+        { -1000.0f, 1000.0f, 0.0f },  { 0.0f, 0.0f, 0.0f },
+    };
+
+    EnDekunuts* this = THIS;
+    s32 i;
+    Vec3f* ptr1;
+    Vec3f* ptr2;
+    s32 value = D_808BEF98[limbIndex];
+
+    if (value != -1) {
+        if (value < 3) {
+            SysMatrix_GetStateTranslationAndScaledX(1000.0f, &this->unk_21C[value]);
+        } else {
+            SysMatrix_GetStateTranslation(&this->unk_21C[value]);
+            ptr1 = &D_808BEFA4[0];
+            ptr2 = &this->unk_21C[value + 1];
+            for (i = value + 1; i < ARRAY_COUNT(this->unk_21C); i++) {
+                SysMatrix_MultiplyVector3fByState(ptr1, ptr2);
+                ptr1++, ptr2++;
+            }
+        }
+    }
+
+    if (limbIndex == 2) {
+        SysMatrix_GetStateTranslation(&this->actor.focus.pos);
+    }
+}
+
+void EnDekunuts_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    EnDekunuts* this = THIS;
+
+    SkelAnime_Draw(globalCtx, this->skelAnime.skeleton, this->skelAnime.limbDrawTbl, func_808BEBD0, func_808BED30,
+                   &this->actor);
+    SysMatrix_SetStateRotationAndTranslation(this->actor.home.pos.x, this->actor.home.pos.y, this->actor.home.pos.z,
+                                             &this->actor.home.rot);
+    Matrix_Scale(this->actor.scale.x, this->actor.scale.y, this->actor.scale.z, MTXMODE_APPLY);
+    if (this->actor.colorFilterTimer != 0) {
+        func_800AE5A0(globalCtx);
+    }
+    func_800BDFC0(globalCtx, D_06001E50);
+    func_800BE680(globalCtx, &this->actor, &this->unk_21C[0], 8, this->unk_214, this->unk_218, this->unk_210,
+                  this->unk_18E);
```

Comment:
```suggestion
    func_800BE680(globalCtx, &this->actor, this->unk_21C, 8, this->unk_214, this->unk_218, this->unk_210,
                  this->unk_18E);
```
This is similar to an above comment. Minislime defines this function as:
`void func_800BE680(GlobalContext* globalCtx, Actor* actor, Vec3f limbPos[], s32 arg3, f32 arg4, f32 arg5, f32 arg6, u8 mode);`